add "payee-meta" configuration option to store payee as metadata

zacchiro opened the pull request:
When set, this overrides the default behaviour of using the "payee | narration"
hledger syntax.

tbm reviewed (approved):
This look good to me.

I'm not sure how much config options to add to b2l.  Originally there weren't any.  But I added some, so I guess it's ok to add more as long as they make sense (which I think this one does).

Can you add your name to the spell check file so the tests work.

zacchiro committed:
add "payee-meta" configuration option to store payee as metadata

When set, this overrides the default behaviour of using the "payee | narration"
hledger syntax.

zacchiro commented:
On Sun, Aug 07, 2022 at 11:01:18PM -0700, Martin Michlmayr wrote:
> Can you add your name to the spell check file so the tests work.

Done, thanks.
